data/systems/custom/07_1orionis.lua

```diff
@@ -131,7 +131,7 @@ local saorsa = {
 		:ice_cover(f(467,1000))
 		:life(f(45,100)),
 	{
-		CustomSystemBody:new('New Edinburgh', 'STARPORT_SURFACE')
+		CustomSystemBody:new('Dùn Éideann Ùr', 'STARPORT_SURFACE')
```

Comment:
I'm of two minds about this one - I would expect New Edinburgh to be a legitimate placename for this system, but I'm also okay with Dun Eideann. I'll leave that up to your discretion.

Reply:
That depends on whether we want to make this a multilingual Scots + Gaelic society, or a monolingual Gaelic society. I'd be fine with either and could research the Scots names for the lowlands-based place names.

Reply:
I think multilingual works a bit better as we don't have a system that's entirely lowland Scots. I'd imagine that even a New Scotland system would be a bit of a cultural melting pot for the various regional Scots languages.